src/sentry/static/sentry/app/components/bases/pluginComponentBase.jsx

```diff
@@ -46,7 +46,7 @@ class PluginComponentBase extends React.Component {
     };
   }
 
-  componentWillMount() {
+  componentDidMount() {
     this.api = new Client();
   }
```

Comment:
Does it make sense to initialize API client in the constructor? There's no benefit for it to be placed in any component lifecycle methods.

Reply:
Yeah probably makes more sense